docs/en/ingest-management/fleet/add-fleet-server-kubernetes.asciidoc

```diff
@@ -0,0 +1,392 @@
+[[add-fleet-server-kubernetes]]
+= Deploy Fleet Server on Kubernetes
+
+[NOTE]
+====
+For ECK users, if your {stack} is orchestrated by ECK, we recommend to orchestrate the {fleet-server} through the operator. That simplifies the process as all needed resources will be configured by ECK.
++
+Refer to {eck-ref}/k8s-elastic-agent.html[Run {agent} on ECK] for more information.
+====
+
+[IMPORTANT]
+====
+This guide assumes certain knowledge about deploying applications on Kubernetes.
+====
+
+To use {fleet} for central management, a <<fleet-server,{fleet-server}>> must
+be running and accessible to your hosts.
+
+You can deploy {fleet-server} on Kubernetes and manage it yourself.
+In this deployment model, you are responsible for high-availability,
+fault-tolerance, and lifecycle management of the {fleet-server}.
+
+To deploy a {fleet-server} on Kubernetes and register it into Fleet you will need the following details:
+
+* The *Policy ID* of a Fleet policy configured with the {fleet-server} integration.
+* A *Service token*, that you can generate following the <<add-fleet-server-kubernetes-preparations>> section, or manually using the {ref}/service-tokens-command.html[elasticsearch-service-tokens] command.
```

Comment:
I would consider removing the links from this bullet point. Why? You don't link out in any of the other bullet points—instead, you use this section to explain what the user needs and why. This bullet point is a bit different; you explain that a service token is needed, but instead of explaining why or what it is, you mention the two methods of creating one. I think it would be better to say something like:

* A *Service token*, used to authenticate Fleet Server with Elasticsearch.

You can then explain the two methods of generating a token in the section below.

Reply:
makes sense! and it was redundant actually. thanks!